From b81cce2918e6b4e4fc5543bac709483d9a88fcfe Mon Sep 17 00:00:00 2001
From: KernelJ <kernelj@epixxware.com>
Date: Sun, 18 Jan 2015 16:38:09 +0000
Subject: [PATCH] Remove -L switch to file in libtool

---
 build-aux/ltmain.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build-aux/ltmain.sh b/build-aux/ltmain.sh
index 0dd6f3a..0cb4a13 100644
--- a/build-aux/ltmain.sh
+++ b/build-aux/ltmain.sh
@@ -2569,7 +2569,7 @@ func_win32_libid ()
 {
   $opt_debug
   win32_libid_type="unknown"
-  win32_fileres=`file -L $1 2>/dev/null`
+  win32_fileres=`file $1 2>/dev/null`
   case $win32_fileres in
   *ar\ archive\ import\ library*) # definitely import
     win32_libid_type="x86 archive import"
-- 
2.1.0

