From 59f6b0b0711930c0d03a5224e15ba64f489263b1 Mon Sep 17 00:00:00 2001
From: KernelJ <kernelj@epixxware.com>
Date: Sun, 18 Jan 2015 18:35:56 +0000
Subject: [PATCH 4/4] Remove -L switch to file in libtool

---
 ltmain.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ltmain.sh b/ltmain.sh
index 499e473..33589b3 100644
--- a/ltmain.sh
+++ b/ltmain.sh
@@ -3543,7 +3543,7 @@ func_win32_libid ()
 {
   $opt_debug
   win32_libid_type="unknown"
-  win32_fileres=`file -L $1 2>/dev/null`
+  win32_fileres=`file $1 2>/dev/null`
   case $win32_fileres in
   *ar\ archive\ import\ library*) # definitely import
     win32_libid_type="x86 archive import"
-- 
2.1.0

