From ff735acf35757b8e178073f329022e519e412c25 Mon Sep 17 00:00:00 2001
From: KernelJ <kernelj@epixxware.com>
Date: Sun, 18 Jan 2015 15:16:25 +0000
Subject: [PATCH 7/7] Remove -L switch to file in libtool

---
 ltmain.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ltmain.sh b/ltmain.sh
index 199b772..4b87d9a 100644
--- a/ltmain.sh
+++ b/ltmain.sh
@@ -205,7 +205,7 @@ func_mktempdir ()
 func_win32_libid ()
 {
   win32_libid_type="unknown"
-  win32_fileres=`file -L $1 2>/dev/null`
+  win32_fileres=`file $1 2>/dev/null`
   case $win32_fileres in
   *ar\ archive\ import\ library*) # definitely import
     win32_libid_type="x86 archive import"
-- 
2.1.0

